modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1911,6 +1911,16 @@ return view.extend({
 				]).then(this.renderContents.bind(this));
 			};
 
+			if (bss.ssid == null) {
+				name = s2.option(form.Value, 'ssid', _('WiFi SSID'), _('SSID must be manually set when a Hidden WiFi is used.'));
+				name.validate = function(section_id, value) {
```

Comment:
I think you can simply replace the entire `name.validate` function with `name.optional = false` or `name.rmempty = false`